plugins/Terminal/js/components/seedprompt.js

```diff
@@ -22,6 +22,8 @@ export default class WalletSeedPrompt extends React.Component {
 					'dictionary': 'english',
 				}))
 				siac.end()
+				e.target.value = ''
```

Comment:
Since I am only adding or removing class attributes instead of changing the dom the value is stored unless I manually clear it.